# Funciones para modelar la relación entre variables

#### Ejercicio 3: 

En Estados Unidos se suele usar la escala Fahrenheit para medir la temperatura. En esta escala, el punto de congelación del agua se alcanza a los $32^\circ F$ y el de ebullición a los $212^\circ F$. El resto del mundo usa la escala Celsius en la que esos puntos se alcanzan a los $0^\circ C$ y $100^\circ C$, respectivamente. Hallar el modelo que relaciona ambas escalas.

#### Solución:

* $F$: escala Fahrenheit --------- variable dependiente.
* $C$: escala Celsius ---------- variable independiente

* $A(0,32)$
* $B(100,212)$

Suponemos que se trata de una función lineal:

$$ F(C)=mC+b $$

$$ m=\dfrac{F_2-F_1}{C_2-C_1} $$

$$ m=\dfrac{212-32}{100-0}=\dfrac{9}{5}=1.8 $$

$$ F(C)=1.8C+b $$

$$ 212-1.8(100)=b=32 $$

$$ F(C)=1.8C+32 $$

$$ F(C)=\dfrac{9}{5}C+32 $$

# Clase 4

## Funciones cuadráticas

$$ f(x)=ax^2+bx+c $$

Al igual que en las funciones lineales, el valor de $c$ lo podemos conocer al saber qué sucede en $x=0$. Si no tenemos este conocimiento, extrapolamos de los datos que tengamos.

Para una función polinomial (lineal, cuadrática, cúbica,...) el dominio máximo de la función $(-\infty,\infty).$

### Gráfica de una función cuadrática

Para las funciones cuadráticas

$$ f(x)=ax^2+bx+c $$

tendremos un punto importante conocido como vértice y que tiene coordenadas

$$ V\left(-\dfrac{b}{2a},c-\dfrac{b^2}{4a}\right) $$

**1er caso:** $a>0$: parábola que abre hacia arriba (tiene un valor mínimo)

Entonces el rango de la función es

$$ \left[c-\dfrac{b^2}{4a},\infty\right) $$

**2do caso:** $a<0$: parábola que abre hacia abajo (tiene un valor máximo)

Entonces el rango de la función es

$$ \left(-\infty,c-\dfrac{b^2}{4a}\right] $$

### Forma de la función cuadrática

Si sabemos que la función cuadrática pasa por el eje de horizontal en dos puntos ($B$ y $C$), podemos decir que la forma de la función cuadrática es

$$ f(x)=A(x-B)(x-C) $$

## Situación. Gravedad en Marte

La exploración de Marte ha requerido del uso de nuevas estrategias a fin de superar
obstáculos con los que se han enfrentado los vehículos destinados a navegar por el
planeta. Algunos de los retos son: el conseguir que los vehículos exploradores
recorran más terreno en menos tiempo, evitando el problema de corrosión de las
ruedas, las tormentas de arena que bloquean los paneles solares y el cambio en las
estaciones del clima en Marte que modifican la incidencia de rayos solares sobre
los paneles.

Para resolver el problema de cubrir más área explorada en menos tiempo, se han
propuesto nuevas ideas como el uso de helicópteros en este planeta o robots
denominados PUH (por sus siglas en inglés) que significa "Saltador Urbano de
Precisión", robots que son capaces de saltar, ahorrando energía en comparación
con volar.

La función que describe el lanzamiento vertical de un objeto como función del
tiempo, es de gran utilidad al momento de determinar la posición del robot PUH
dada su altura inicial, en cualquier cuerpo celeste del que se conoce su gravedad.
Cuando un objeto se lanza hacia arriba desde una altura inicial $h_0$, con una
velocidad inicial $v_0$, la función para determinar su altura es $H(t)$ en el tiempo $t$ está dada por

$$H(t)=h_0+ v_0t −\dfrac{1}{2}gt^2$$

donde $g$ es el valor de la aceleración gravitacional de Marte.

En el esquema de la figura se ilustra el salto en vertical de un robot desde
una altura inicial $h_0$, con una velocidad inicial $v_0$, que alcanza una altura máxima $H(t_\text{max})=\text{max}$, en el tiempo $t_\text{max}$ , para después caer a la superficie del planeta, alcanzando así una altura final $H(t_f)=0$, en el tiempo final $t_f$.


<img src="marte1.png" align="middle" width="400">

**1. El modelo que describe la altura del objeto en lanzamiento vertical como función
del tiempo es una:**

A) Función de primer grado, $f(x)=ax+b$

B) Función de segundo grado $f(x)=ax^2+bx+c$

C) Función trigonométrica $f(x)=A\sin(bx)+c$

D) Función exponencial $f(x)=Ba^x+c$

**Solución:** B)

**2. ¿Cuál es el dominio máximo de definición de la función?**


A) $(-\infty,\infty)$

B) $[0,15)$

C) $(0,\infty)$

D) $[0, 360)$

**Solución:** A)

**3. Si en el planeta Marte, desde una plataforma situada a una altura $h_0=1.5\ \rm{m}$ un robot salta verticalmente con una velocidad inicial $v_0=20\ \dfrac{\rm{m}}{\rm{s}}$ y al considerar que la aceleración gravitacional en Marte es de $g=3.71\ \dfrac{\rm{m}}{\rm{s}^2}$.**

**¿Cuál es la altura en metros, que el robot alcanza a los dos segundos de su lanzamiento hacia arriba?**

$$H(t)=h_0+ v_0t −\dfrac{1}{2}gt^2$$

* Datos:
    * $h_0=1.5$
    * $v_0=20$
    * $g=3.71$
    * $H=?$
    * $t=2$

$$H(t)=1.5+20t −1.855t^2$$

$$H(2)=1.5+20(2) −1.855(2)^2$$

In [2]:
1.5+20*(2)-1.855*(2)^2

34.08

A) 45.92

B) 34.08

C) 31.08

D) 48.92

**Solución:** B)

**4. Considerando las circunstancias en la pregunta anterior ¿Cuál es el tiempo
aproximado, en segundos en el que el robot toca la superficie del planeta después
de su salto vertical desde la plataforma?**

<img src="marte1.png" align="middle" width="400">

$$H(t)=1.5+20t −1.855t^2$$

Si el robot llega hasta el piso entonces la altura final es cero.

$$0=1.5+20t −1.855t^2$$

$$ at^2+bt+c=0 $$

$$ t=\dfrac{-b\pm\sqrt{b^2-4ac}}{2a} $$

$$ t_1=\dfrac{-b+\sqrt{b^2-4ac}}{2a}=-0.07 $$

In [3]:
(-(20)+√(20^2-4(-1.855)(1.5)))/(2*(-1.855))

-0.07448541584218686

$$ t_2=\dfrac{-b-\sqrt{b^2-4ac}}{2a}=10.85 $$

In [4]:
(-(20)-√(20^2-4(-1.855)(1.5)))/(2*(-1.855))

10.856156574871836

$a=-1.855$, $b=20$, $c=1.5$

A) 5.46

B) 5.31

C) 10.70

D) 10.856

**Solución:** D)

**5. De las siguientes gráficas, ¿cuál representa un esbozo de la gráfica de la función correspondiente al lanzamiento vertical descrito en la pregunta 3?**

$$H(t)=1.5+20t −1.855t^2$$

$$H(0)=1.5+20(0) −1.855(0)^2$$

C)

$(0,0)$

D)

$(0,1.5)$

C)
<img src="cuadratica5C.png" align="middle" width="400">

D)
<img src="cuadratica5D.png" align="middle" width="400">

**Solución:** D)

**6. La figura muestra la gráfica de la expresión matemática que describe la altura del robot en función del tiempo, cuando éste realiza un salto vertical desde la luna, donde hay una aceleración gravitacional $g=1.6\ \dfrac{\rm{m}}{\rm{s}^2}$ , inicia dicho salto a nivel del suelo y a una velocidad inicial de $v_0=40\ \dfrac{\rm{m}}{\rm{s}}$.**

**¿Cuáles son los puntos A y B de dicha gráfica?**


* Datos:
    * $h_0=0$
    * $v_0=40$
    * $g=1.6$

$$H(t)=40t −0.8t^2$$

$$0=40t −0.8t^2$$

In [5]:
40*(50)-0.8*(50)^2

0.0

In [6]:
40*(25)-0.8*(25)^2

500.0

$$0=t(40−0.8t)$$

$t=0$

$40-0.8t=0$

$t=\dfrac{40}{0.8}$

<img src="marte2.png" align="middle" width="500">

B) $A(0,0),\ B(50,0)$

C) $A(0,0),\ B(25,0)$

**Solución:** B)

**7. De acuerdo con la información de la gráfica asociada con la gráfica de la figura anterior, ¿cuál es el dominio natural de la función?**

Dominio $\leftrightarrow$ abscisas

Rango $\leftrightarrow$ ordenadas

A) $[0, 25]$

B) $[25, 0]$

C) $[0, 50]$

D) $[50, 0]$

**Solución:** C)

**8. La siguiente figura muestra la gráfica de la expresión matemática que describe la
altura del robot en función del tiempo, cuando éste realiza un salto vertical desde la
luna, e inicia dicho salto a nivel del suelo.**

**¿Cuál es la expresión matemática que modela la gráfica de la figura?**

<img src="marte3.png" align="middle" width="500">

A) $H ( t ) = 0.8( t − 0)(t − 62.5)$

B) $H (t) = − 0.8(t − 0)(t − 62.5)$

C) $H (t) = − 0.8(t + 0)(t + 62.5)$

D) $H ( t ) = 0.8( t + 0)(t + 62.5)$

**Solución:** B)

**9. La siguiente figura corresponde a la gráfica de la altura de un robot lanzado en la luna, desde una alura de $1.2\ \rm{m}$, como función del tiempo ¿Cuál es la expresión que corresponde a dicha gráfica?**


<img src="marte4.png" align="middle" width="500">

**Solución:** 

**10. ¿Cuál es el recorrido (rango) de la función asociado con el lanzamiento vertical de un robot en el planeta Marte a una altura inicial de $2.145\ \rm{m}$, con una velocidad inicial de $3.71\ \dfrac{\rm{m}}{\rm{s}}$, al considerar el dominio máximo de la misma?**

A) $(-\infty, 4]$

B) $(1, \infty)$

C) $[1, \infty)$

D) $(-\infty, 4)$

**Solución:** 

**11. ¿Cuál es el recorrido (rango) de la función considerando el dominio natural de la misma, para la situación planteada en la pregunta anterior?**

A) $[4,\infty)$

B) $[-\infty,4)$

C) $(0,4)$

D) $[0,4]$

**Solución:** 

## Logaritmos

* ***Ejemplo:*** Resuelve la ecuación

$$ 3^{x+1}=81 $$

* ***Ejemplo:*** Resuelve la ecuación

$$ \log_2(x^7)=21 $$

* ***Ejemplo:*** Resuelve la ecuación

$$ \log_2(x+1)+\log_2(x-1)=4 $$

* ***Ejemplo:*** Resuelve la ecuación

$$ \log_2(x+1)+\log_2(x-1)=4 $$

## Funciones exponenciales

$$ f(x)=a\cdot b^x+c $$

* Si $b>1$, decimos que se trata de un crecimiento exponencial.
* Si $b<1$, decimos que se trata de un decaimiento exponencial.
* Si $b=1$...

El dominio máximo de estas funciones es $\mathbb{R}$.

* ***Ejemplo:*** Pensemos que tenemos una función exponencial $f(x)$. Si $f(20)=88.2$ y $f(23)=91.4$, encuentra la (a) base, y (b) $f(25)$.

## Funciones logarítmicas

$$ f(x)=a\cdot\log(x+b)+c$$

## Situación: Fragmentos de Bosque y Hormigas

Como parte de un análisis del efecto de la fragmentación sobre las relaciones
especies-área, se registró el número de especies de hormigas para ciertos
fragmentos de bosque seco tropical. En el siguiente gráfico se presenta la relación
entre el área de los fragmentos de bosque seco tropical en la cuenca media del río
Cauca, Colombia, medida en hectáreas (h) , con el número de especies de hormigas
encontradas en cada fragmento N (h) .


**1. Es el tipo de función que mejor describe la asociación entre el área y el número
de especies que se representa en la figura.**

A) $N(h)=13e^h+76$

B) $N(h)=76e^h+13$

A) $N(h)=13\log(h)+76$

D) $N(h)=76\log(h)+13$

**Solución:** 

## ¿Quién gana?